src/App.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {Component} from 'react';
+import Navbar from './components/Navbar/Navbar';
+import UserBoard from './components/UserInput/UserBoard';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null
+    }
+  }
+
+  fetchData = () => {
```

Comment:
This could be a great example of a place to use another component! Then you could drop this api fetch in wherever you want!